python/ray/autoscaler/ray-schema.json

```diff
@@ -307,6 +307,10 @@
         "no_restart": {
             "description": "Whether to avoid restarting the cluster during updates. This field is controlled by the ray --no-restart flag and cannot be set by the user."
         },
+        "no_config_validation_warning": {
```

Comment:
Do we need to make this a flag? It seems fine to just turn this to a warning by default globally

Reply:
it is a warning by default globally. This is disabling the warning altogether. Reason I was thinking is showing a warning on the monitor logs for users on the product is weird when they have no control of what version of ray is used in the product api.

The naming would probably be better as `cluster_config_validation_disable_warning`.

Reply:
If the concern is about adding too many flags, we could just disable the warning altogether.

Reply:
I see, how about we don't show the warning then?

Reply:
done